packages/web/docs/src/content/management/access-tokens.mdx

```diff
@@ -0,0 +1,180 @@
+import NextImage from 'next/image'
+import { Callout, Steps } from '@theguild/components'
+
+# Access Tokens
+
+Access Tokens in GraphQL Hive are secure credentials used to authenticate API requests and automate
+actions without using personal login details.
+
+With Access Tokens, you can perform tasks such as:
+
+- Publish schemas etc. with the Hive CLI
+- Report usage data (SDK or HTTP)
+- (Soon) Access the public Hive Console GraphQL API for managing resources and export data
```

Comment:
I don't love having future work in docs. I think let's represent what we have and we could link to our issues to show upcoming work.